depends/Makefile

```diff
@@ -1,7 +1,7 @@
 CONFIG = ../Options.mk
 include $(CONFIG)
 
-LIBRARIES=lib/libbigfile-mpi.a lib/libmpsort-mpi.a lib/libpfft_omp.a
+LIBRARIES=lib/libbigfile-mpi.a lib/libmpsort-mpi.a lib/libpfft_omp.a lib/libfftw3_mpi.a lib/libfftw3_omp.a
```

Comment:
Do we link against these fftw3 libraries explicitly?

Reply:
Yes - I took the list from Makefile.rules.